markdown/en/api/ff-compare.api.md

```diff
@@ -3,12 +3,12 @@ ___
 ### Properties
 | Name | Description |
 | ---- | ----------- |
-|**record-id** (Array) |The record Id's of the products to compare.|
-|**max-results** (Number) (default:4)|The maximum amount of dispatched records.|
-|**subscribe** (String) **Options**: true, false (default:true)| If this is set to `false` the element no longer listens to events and therefore does not receive new data. This is useful if you want to dispatch your own data to the element.|
-|**auto-compare** (String) **Options**: true, false (default:false)|When set to true, the request to the compare service will be automaticaly triggered when the attribute 'record-id' are changed.|
+| **record-id** (Array) | The record IDs of the products to compare. |
+| **max-results** (Number) (default:4) | The maximum amount of dispatched records. |
+| **subscribe** (String) **Options**: "true", "false" (default:"true") | If this is set to `"false"` the element no longer listens to events and therefore does not receive new data. This is useful if you want to dispatch your own data to the element. |
+| **auto-compare** (String) **Options**: "true", "false" (default:"false") | When set to `"true"`, the request to the compare service will be automatically triggered when the attribute `record-id` are changed. |
```

Comment:
When set to `"true"`, _(I think no comma)_ the request to the compare service will be automatically triggered when the attribute `record-id` ~~are~~ _is_ changed. |

Reply:
[Most definitely a comma.](https://www.grammarly.com/blog/conditional-sentences/)
Indeed, *is*.

Reply:
Yup! That needs comma.